jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/RetrieveRegistryCredentialsStep.java

```diff
@@ -82,21 +85,21 @@ static RetrieveRegistryCredentialsStep forTargetImage(
   @Override
   @Nullable
   public Credential call() throws CredentialRetrievalException {
-    buildLogger.lifecycle(String.format(DESCRIPTION, registry) + "...");
+    String description = makeDescription(registry);
+    eventEmitter.emit(LogEvent.lifecycle(description + "..."));
```

Comment:
Why not make the "..." part of `makeDescription`? Actually, `makeDescription` seems simple enough to just inline here?

Reply:
The description without the "..." is passed into the `TimerEventEmitter`. I have `makeDescription` separate mostly to have it sort of like a static constant up top.